Core/include/Acts/EventData/ParticleHypothesis.hpp

```diff
@@ -44,10 +44,12 @@ class SinglyChargedParticleHypothesis
   static SinglyChargedParticleHypothesis electron() {
     return SinglyChargedParticleHypothesis(PdgParticle::eElectron);
   }
+  static SinglyChargedParticleHypothesis kaon() {
+    return SinglyChargedParticleHypothesis(PdgParticle::eKaon);
+  }
   static SinglyChargedParticleHypothesis proton() {
     return SinglyChargedParticleHypothesis(PdgParticle::eProton);
   }
-
```

Comment:
I would keep the separation between real particles and test particles

Reply:
Corrected